docker-compose.yml

```diff
@@ -50,3 +55,11 @@ services:
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock:ro"
       - "./etc/data/portainer:/data"
+  dnsmasq:
+    build: .
```

Comment:
For local builds, put Dnsmasq Dockerfile into a directory b/dnsmasq/ and change build: to build: b/dnsmasq, this avoids renaming the dockerfile.